src/Naming.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+
+export default class Naming extends Component {
+  render () {
+    return (
+      <div className='Naming' >
+          You will lead your team on the trail.
```

Comment:
These should probably be in `<p>` tags.

Reply:
Putting `<br>` between paragraphs is pretty trailer-park. You should accomplish this with CSS.